From bb876407b553af0fd55b2b07f0261d1f02eae6c1 Mon Sep 17 00:00:00 2001
From: Jo Henke <37883863+jo-he@users.noreply.github.com>
Date: Tue, 6 Apr 2021 11:38:28 +0000
Subject: [PATCH] dwc-otg: fix clang -Wsometimes-uninitialized warning

warning: variable 'retval' is used uninitialized whenever 'if' condition is false
---
 drivers/usb/host/dwc_otg/dwc_otg_cil_intr.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/host/dwc_otg/dwc_otg_cil_intr.c
+++ b/drivers/usb/host/dwc_otg/dwc_otg_cil_intr.c
@@ -805,7 +805,7 @@ static int32_t dwc_otg_handle_pwrdn_sess
  */
 static uint32_t dwc_otg_handle_pwrdn_stschng_intr(dwc_otg_device_t *otg_dev)
 {
-	int retval;
+	uint32_t retval = 0;
 	gpwrdn_data_t gpwrdn = {.d32 = 0 };
 	gpwrdn_data_t gpwrdn_temp = {.d32 = 0 };
 	dwc_otg_core_if_t *core_if = otg_dev->core_if;
